lib/openstax/aws/stack.rb

```diff
@@ -18,6 +19,16 @@ def initialize(id: nil, name:, region:, enable_termination_protection: false,
       raise "Stack name must not be blank" if name.blank?
       @name = name
 
+      raise "`tags` must be a hash" if !tags.is_a?(Hash)
```

Comment:
this might look better using an unless.   do we really need to check the type tho?  checking type is a bit anti-ruby-ish

Reply:
I hate `unless` :-) 

Type checking seems ok - alternative is someone passes an array and then the code will blow up on line 25 with some "No [] method for array".